build/images/base/Dockerfile

```diff
@@ -57,3 +76,4 @@ RUN apt-get update && apt-get install -y --no-install-recommends ipset jq inotif
     /iptables-wrapper-installer.sh
 
 COPY --from=cni-binaries /opt/cni/bin /opt/cni/bin
+COPY --from=gobgp-binary /opt/bgp/gobgp /opt/bgp/gobgp
```

Comment:
I don't think this is in the PATH, shouldn't we place the binary in one of the path directories so it can be invoked directly?
Note that things are different for CNI binaries: these are just destined to be copied to the host, they are not invoked directly from the container.

Reply:
For invoking directly we would need to keep it in ENV PATH /usr/local/bin